pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -335,6 +335,7 @@ func NewContainerManager(mountUtil mount.Interface, cadvisorInterface cadvisor.I
 	cm.topologyManager.AddHintProvider(cm.cpuManager)
 
 	cm.memoryManager, err = memorymanager.NewManager(
+		ctx,
```

Comment:
Would passing `context.TODO()` here reduce amount of changes in `pkg/kubelet/cm`?

Reply:
Will look into this, Thanks!

Reply:
PTAL https://github.com/kubernetes/kubernetes/pull/130727/commits/c7989a1ab6bb31533939deec9b6567be76accb2a and let me know your thoughts. Thanks!

Reply:
I think it looks better. There were a lot more changes in pkg/kubelet/cm before as far as I remember.